PWGHF/HFL/Tasks/taskElectronWeakBoson.cxx

```diff
@@ -72,6 +75,21 @@ struct HfTaskElectronWeakBoson {
   Configurable<float> energyIsolationMax{"energyIsolationMax", 0.1, "isolation cut on energy"};
   Configurable<int> trackIsolationMax{"trackIsolationMax", 3, "Maximum number of tracks in isolation cone"};
 
+  struct HfElectronCandidate {
+    float pt, eta, phi, energy;
+    ElectronCandidate(float p, float e, float ph, float en)
+      : pt(p), eta(e), phi(ph), energy(en) {}
+
+    // 運動量成分を計算する関数を追加
+    float px() const { return pt * std::cos(phi); }
+    float py() const { return pt * std::sin(phi); }
+    float pz() const { return pt * std::sinh(eta); }
+  };
+  std::vector<HfElectronCandidate> selectedElectronsIso;
+  std::vector<HfElectronCandidate> selectedElectronsAss;
+
+  HfHelper hfHelper;
+
```

Comment:
This seems completely unnecessary if you just use `std::array` for the momenta and `RecoDecay` methods.